features/step_definitions/_given_steps.rb

```diff
@@ -233,12 +233,12 @@
 Given /^the [pP]roject(?: "([^\"]*)")? has the following impediments:$/ do |project_name, table|
   project = get_project(project_name)
 
-  User.current = User.find(:first)
+  User.current = User.first
 
   as_admin do
     table.hashes.each do |impediment|
-      sprint = Sprint.find(:first, conditions: { name: impediment['sprint'] })
-      blocks = Story.find(:all, conditions: { subject: impediment['blocks'].split(', ')  }).map(&:id)
+      sprint = Sprint.find_by(name: impediment['sprint'])
+      blocks = Story.where(subject: impediment['blocks'].split(', ')).pluck(:id)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.